releasenotes/notes/fix-context-serializable-2c9768cbe4738b73.yaml

```
@@ -0,0 +1,4 @@
+---
+fixes:
+  - |
+    tracing: fix manual tracing across processes, make ``ddtrace.context.Context`` pickable.
```

```suggestion
    tracing: fix manual tracing across processes, make ``ddtrace.context.Context`` serializable.
```

"pickable" (should probably be "pickleable"?) isn't a real word.